src/main/java/jdbchomework/utils/TestUtils.java

```diff
@@ -0,0 +1,20 @@
+package jdbchomework.utils;
+
+import jdbchomework.entity.Company;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class TestUtils {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)